workflow/lib/sbs_process/configuration_utils.py

```diff
@@ -5,38 +5,38 @@
 from lib.shared.configuration_utils import outline_mask
 
 
-def annotate_segment_on_sequencing_data(data, nuclei, cells):
+def annotate_segment_on_sequencing_data(cellpose_rgb, nuclei, cells):
     """Annotate outlines of nuclei and cells on sequencing data.
 
```

Comment:
Why do we need two annotate on sequencing data functions?

Reply:
Done in https://github.com/cheeseman-lab/brieflow/pull/26/commits/58e90e8cbcdb7939bac941d4192d58719076e46b